python/instrumentation/openinference-instrumentation-agno/tests/test_instrumentor.py

```diff
@@ -110,4 +109,4 @@ def test_agno_instrumentation(
             assert attributes.get("llm.model_name") == "gpt-4o-mini"
             assert attributes.get("llm.provider") == "OpenAI"
             assert span.status.is_ok
-    assert checked_spans == 4
+    assert checked_spans == 2
```

Comment:
I think the tests might be flaky due to the duckdockgo tool. Might want to move off of a tool that requires a network call.